biome.json

```diff
@@ -22,6 +22,7 @@
         "useHookAtTopLevel": "error",
         "useIsNan": "error",
         "noUnusedPrivateClassMembers": "error",
+        "noUnusedImports": "error",
```

Comment:
Did you mean to change this here? Or was it to help with the codemod

Reply:
Nah, I'm removing these, I just set them so that biome could go and fix them. I'm going to go through all the rules to make sure nothing is borked.